boards/nxp/frdm_mcxn947/frdm_mcxn947.dtsi

```diff
@@ -186,5 +186,21 @@
 
 &lpadc0 {
 	pinctrl-0 = <&pinmux_lpadc0>;
+};
+
+nxp_flexio_lcd: &flexio0_lcd {
+	/* DMA channels 0, muxed to FlexIO TX */
+	dmas = <&edma0 0 61>;
+	dma-names = "tx";
+	shifters_count = <8>;
+	timers_count = <1>;
+	enwr-pin = <1>;
+	rd-pin = <0>;
+	data-pin-start = <16>;
+	data-bus-width = <16>;
```

Comment:
Is there plan to support 8-bit data width? if yes, then need to define a macro `FLEXIO_MCULCD_DATA_BUS_WIDTH`:
https://github.com/zephyrproject-rtos/hal_nxp/blob/abc66979c77421fb3a140ce2e4e6ea7165cdbe8f/mcux/mcux-sdk/drivers/flexio/mculcd/fsl_flexio_mculcd.h#L44C1-L49C3

Reply:
Yes, this is passed as a compile time MACRO via the CMake file. I have added a change request to the SDK driver to avoid using this macro as it has limitations. Please see my change request for details